tests/test_filters.py

```diff
@@ -404,15 +440,22 @@ def _assert_column_definition(
         column_name,
         column_type,
         column_type_length,
-        column_attributes
+        column_attributes,
+        column_type_values=None
```

Comment:
Not really a fan having all the parameters mixed together and then checking some conditions to decide which parameters will be verified. The better way to do this is to create a separate function specifically for the column definition with values. This existing function is then specifically for the column definition with length.